backend/src/api/users/users.controller.ts

```diff
@@ -23,4 +23,15 @@ export class UsersController {
     const statusCode = user ? HttpStatus.OK : HttpStatus.NOT_FOUND;
     res.status(statusCode).send(user);
   }
+
+  @Get(':userId/profile-image')
+  @UseGuards(AuthenticatedGuard)
+  async downloadProfileImage(
```

Comment:
```suggestion
  async getProfileImage(
```
download보다 get이 좋지 않을까요? 🧐  큰 문제는 아님..